static/js/components/B2BReceiptExplanation.js

```diff
@@ -0,0 +1,29 @@
+// @flow
+import React from "react"
+
+import Expandable from "./Expandable"
+
+type Props = {
+  className?: string
+}
+const B2BReceiptExplanation = ({ alreadyPaid, className }: Props) => (
```

Comment:
A few concerns here about `alreadyPaid`

- If you have to use it, you need to add it in the types too
- It is not being used anywhere so it should not be there